e2e/src/form-input/form-input.po.ts

```diff
@@ -48,15 +50,31 @@ export class FormInputPage {
   // [for="radio-4"]
   private readonly CSS_SELECTOR_LABEL_RADIO_TYPE_NUMBER = this.getLabelForAttribute(this.ID_RADIO_TYPE_NUMBER);
 
-  // [for="radio-6"]
-  private readonly CSS_SELECTOR_LABEL_RADIO_ICON_NONE = this.getLabelForAttribute(this.ID_RADIO_ICON_NONE);
+  // [for="radio-5"]
+  private readonly CSS_SELECTOR_LABEL_RADIO_TYPE_SEARCH = this.getLabelForAttribute(this.ID_RADIO_TYPE_SEARCH);
 
   // [for="radio-7"]
```

Comment:
È normale si sia perso il `[for="radio-6"]`?

Reply:
Si, è stato aggiunto un nuovo radio button per selezionare il tipo "search" quindi è cambiato l'id